### ISSUE TITLE
dotNet lambda deploy-function exception AWSSDK.SecurityToken

### ISSUE THREAD

**Issue by kcliffe** [aws/aws-lambda-dotnet#223]
We're using the command:
dotnet lambda deploy-function --profile assume_role with AWS Lambda Tools for .NET Core functions (2.1.0)

## Expected Behavior
The lambda function is published to the region deployed in the .aws/config file

## Current Behavior
We receive the exception: 
Error retrieving configuration for function Agrigate_Push: Assembly AWSSDK.SecurityToken could not be found or loaded. This assembly must be available at runtime to use Amazon.Runtime.AssumeRoleAWSCredentials.

## Context
We are trying to deploy a (.NETCore 2.0) lambda 

## Your Environment

* AWSSDK.Core version used: 1.0.0
* Operating System and version: 
* Visual Studio version: 2017 v15.5.4
* Targeted .NET platform: .NET Core 2.0

## .NET Core Info
.NET Command Line Tools (2.1.4)

Product Information:
 Version:            2.1.4
 Commit SHA-1 hash:  5e8add2190

Runtime Environment:
 OS Name:     Windows
 OS Version:  10.0.15063
 OS Platform: Windows
 RID:         win10-x64
 Base Path:   C:\Program Files\dotnet\sdk\2.1.4\

Microsoft .NET Core Shared Framework Host

  Version  : 2.0.5
  Build    : 17373eb129b3b05aa18ece963f8795d65ef8ea54

* Contents of project.json/project.csproj: 
```
<Project Sdk="Microsoft.NET.Sdk">

  <PropertyGroup>
    <TargetFramework>netcoreapp2.0</TargetFramework>
    <GenerateRuntimeConfigurationFiles>true</GenerateRuntimeConfigurationFiles>
  </PropertyGroup>

  <ItemGroup>
    <PackageReference Include="AWSSDK.Securitytoken" Version="3.3.3.3" />
    <PackageReference Include="Amazon.Lambda.Core" Version="1.0.0" />
    <PackageReference Include="Amazon.Lambda.Serialization.Json" Version="1.1.0" />
    <PackageReference Include="flurl" Version="2.6.0" />
    <PackageReference Include="flurl.http" Version="2.1.1" />
  </ItemGroup>

  <ItemGroup>
    <DotNetCliToolReference Include="Amazon.Lambda.Tools" Version="2.1.0" />
  </ItemGroup>

</Project>

```

**Comment by normj** [comment]
Taking a look. I think we just need to add the AWSSDK.Securitytoken reference to Amazon.Lambda.Tools list of dependencies.

**Comment by normj** [comment]
I just released version 2.1.1 of `Amazon.Lambda.Tools` that adds the AWSSDK.SecurityToken dependency to it so assume role will now work.
